### PR TITLE
/a/pages/visualize improvements

### PR DESCRIPTION
**Pull request by eeshugerman**
- closes https://github.com/Requarks/wiki/pull/1634 (adds pan/zoom)
- closes https://github.com/Requarks/wiki/issues/1904
- adds ctrl-click to open a page in a new tab

demo: https://imgur.com/a/vMPlHVB

### CODE REVIEW

File: client/components/admin/admin-pages-visualize.vue [hunk 1]
```diff
@@ -86,41 +93,36 @@ export default {
...
+              [child, _.filter(descendantsGroup, d => d.path != child.path)])
+        return {
+          ...parent,
+          children: descendantsByChild.map(_.partial(recurse, depth + 1))
+        }
       }
+      const root = { path: this.currentLocale, title: this.currentLocale, links: [] }
+      // start at depth=2 because we're taking {locale} as the root and
+      // all paths start with {locale}/
+      return recurse(2, [root, pages])
```
Comment (eeshugerman) [inline client/components/admin/admin-pages-visualize.vue:119]: see https://github.com/Requarks/wiki/issues/1904#issuecomment-631760934

File: client/components/admin/admin-pages-visualize.vue [hunk 2]
```diff
@@ -371,9 +394,11 @@ export default {
 <style lang='scss'>
 .admin-pages-visualize-svg {
   text-align: center;
+  height: 78vh;  // fit to height; no vertical scroll
```
Comment (eeshugerman) [inline client/components/admin/admin-pages-visualize.vue:397]: is there a better way to do this?

Comment (NGPixel) [reply client/components/admin/admin-pages-visualize.vue]: See my commit below.

**Comment by NGPixel** [comment]
Thanks! :+1:

**Pull request merged**